src/deps.ts

```diff
@@ -288,7 +288,7 @@ export interface AutoEncryptionOptions {
    *
    * **NOTE**: Supplying options.schemaMap provides more security than relying on JSON Schemas obtained from the server.
    * It protects against a malicious server advertising a false JSON Schema, which could trick the client into sending decrypted data that should be encrypted.
-   * Schemas supplied in the schemaMap only apply to configuring automatic encryption for client side encryption.
+   * Schemas supplied in the schemaMap only apply to configuring automatic encryption for in use encryption.
```

Comment:
is `schemaMap` applicable to QE? if not, "CSFLE" or "Client-Side Field Level Encryption" would be more appropriate here